controllers/reader_controller.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../modules/validation.rb'
+
+class Reader
+  attr_accessor :name, :email, :city, :street, :house
+  include Validation
+
+  def initialize(name, email, city, street, house)
+    about = [name, email, city, street]
+    about = about.map { |x| check_class(x, String) }
+    return unless (house.is_a? Integer) && about
```

Comment:
too complex condition for unless

Reply:
[name, email, city, street].map { |x| check_class(x, String) }
check_class(house, Integer)